frontend/package.json

```diff
@@ -25,8 +25,8 @@
   },
   "eslintConfig": {
     "extends": [
-      "react-app",
-      "react-app/jest"
+      "",
+      ""
```

Comment:
if this is intentional, let's just remove instead of leaving as empty quotes

Reply:
This is intentional, if I don't do this, it throws the error:
"ESLint couldn't find the config "react-app" to extend from. Please check that the name of the config is correct.
The config "react-app" was referenced from the config file in "...\togetherwewrap\frontend\package.json"."

I will remove the empty quotes!